en/docs/fragments/register-a-service-provider.md

```diff
@@ -1,6 +1,6 @@
 1. Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
 
-2. Navigate to the **Service Providers** tab listed under the **Identity** section and click **Add**.
+2. Navigate to the **Service Providers** tab listed under the **Main** > **Identity** section and click **Add**.
```

Comment:
This seems a little confusing. Shall we say, 

Navigate to **Main** > **Identity** > **Service Providers** > **Add**.